packages/rum-core/src/performance-monitoring/capture-hard-navigation.js

```diff
@@ -232,75 +232,81 @@ function createUserTimingSpans(entries, transactionEnd) {
 
 function captureHardNavigation(transaction) {
   const perf = window.performance
-  if (transaction.isHardNavigation && perf && perf.timing) {
-    const timings = perf.timing
-    if (transaction.marks && transaction.marks.custom) {
-      var customMarks = transaction.marks.custom
-      Object.keys(customMarks).forEach(key => {
-        customMarks[key] += transaction._start
-      })
-    }
+  if (!(transaction.isHardNavigation && perf && perf.timing)) {
```

Comment:
Not sure why negating the condition here would be better that having a simple if around the logic that depends on this condition.